service/src/main/kotlin/xyz/lebkuchenfm/domain/eventstream/Event.kt

```diff
@@ -50,5 +52,10 @@ sealed interface Event {
     data class Say(
         val text: String,
         val audio: Base64EncodedAudio,
-    ) : Event
+    ) : Event {
+        companion object {
```

Comment:
im not sure if I like this here. I mean I like how it looks in commands processors but...

Reply:
Maybe we should create `RadioPersonalityUtterance+SayEvent.kt` file in `xyz.lebkuchenfm.domain.radiopersonality` package 

```
package xyz.lebkuchenfm.domain.radiopersonality

import xyz.lebkuchenfm.domain.eventstream.Event


fun RadioPersonalityUtterance.toSayEvent() = Event.Say(this.text, this.audio)

```

and then use it like 

```
eventStream.sendToEveryone(utterance.ToSayEvent())
```

🤔 

syntax is a bit worse, but separation feels better

Reply:
Separation from what? We're creating an event. This method is just providing another way to create it. It's not really a radio personality thing, or command processor thing or even event stream thing. This is just a part of our domain.